plugins/labhub.py

```diff
@@ -302,11 +302,27 @@ def difficulty_level(user, iss):
                 elif self.GH3_ORG.is_member(user):
                     return True
 
+            @register_check
+            def block_gci_issue_assignment(user, iss):
+                """
+                True if the issue is not labelled with 'initiatives/gci'.
+                False if the issue has been labelled with 'initiatives.gci'.
+                """
+                if 'initiatives/gci' in iss.labels:
+                    return False
+                else:
+                    return True
```

Comment:
This might also be a good time to make a new way to distinguish GCI members and allow them to work on `initiatives/gci` issues. I think the best way to do this is to create a GCI team in the organization and make sure the user is in that team before assigning them to the issue.